### DISCUSSION TITLE
Self congratulatory nonsense

### DISCUSSION THREAD

**Discussion by lookaufo** [ThioJoe/YT-Spammer-Purge#879] [Ideas]
Hi all,

I've recently come across a video on youtube that appears to have comments somewhere in the realm of 95% self congratulatory nonsense. I may have my biases but the video in question is from a youtuber Kyle Hill. I let his video on three mile island autoplay and now I'm down a rabbit hole that I may need some help with. 

https://www.youtube.com/watch?v=cL9PsCLJpAA&t=464s

The video is a snoozefest at best and out of the supposed 580k views there are thousands of positive reviews but virtually no dissenting opinions. Most of the reviews follow a similar pattern of my *relative* was *credential* and what you said is exactly right or  some other pat on the back. 

I suspect these accounts to mostly be bots but even in sensitive smart mode only 12 of these were matches. 

Has any thought been given to expanding into the realm of self congratulatory nonsense bots? I feel like that would be of value to everyone. Also how crazy do I sound right now? haha

[Spam_Log_2022-07-05_17-55-47.txt](https://github.com/ThioJoe/YT-Spammer-Purge/files/9050778/Spam_Log_2022-07-05_17-55-47.txt)

**Comment by Firecul** [comment]
> I suspect these accounts to mostly be bots

How did you come to that conclusion? couldn't those people actually (to use your example) have relatives with knowledge/expertise/experience in that matter?

> Has any thought been given to expanding into the realm of self congratulatory nonsense bots? I feel like that would be of value to everyone. 

If you can think of a way of separating spam comments of that type (I'll be honest I didn't see any at a quick glance) from the legit ones I think we'd all listen

>Also how crazy do I sound right now?

Very 😉
